environment.yml

```diff
@@ -29,10 +29,11 @@ dependencies:
   - numpy
   - h5py
   # --------- geo --------- #
-  - pdal
+  - pdal>=2.6.0
```

Comment:
I guess it's debatable but I prefer to "lock" the version (a "==" instead of a ">=") because otherwise the environment might not work anymore when the next major version is out.

Reply:
updated, I'd keep the minor version free though if it's ok